MetadataExtractor/IO/SequentialStreamReader.cs

```diff
@@ -98,5 +98,10 @@ public override bool TrySkip(long n)
                 return false;
             }
         }
+
+        public override bool IsCloserToEnd(long numberOfBytes)
+        {
+            return _stream.Position + numberOfBytes >= _stream.Length;
```

Comment:
FYI the only reason I haven't merged this yet is because I want to look into this a bit more. Does `Stream.Length` ever throw or cause the stream to read to the end? Not all streams know their lengths. I'm incredibly busy at the moment but will get to this PR along with others that are pending ASAP.

I'm happy for anyone interested in the project to review PRs. It definitely helps improve quality, and I'm all for it.